### PR TITLE
Adding Rotate Anticlockwise button

### PR DESCRIPTION
**Pull request by nisarga-kh**
Issue:
This PR is to add a new type of button called Rotate anticlockwise button. This button rotates in anticlockwise direction when hovered.

### CODE REVIEW

File: website/src/data/buttons.json [hunk 1]
```diff
@@ -284,6 +284,13 @@
                 "type": "ripple",
                 "label": "Ripple",
                 "classes": "ripple-btn"
+            },
+            {
+                "type": "rotate-anticlockwise",
+                "label": "Rotate Anticlockwise",
+                "classes": "rotate-anticlockwise-btn" 
+
+
```
Comment (shahednasser) [inline website/src/data/buttons.json:293]: Remove these empty lines

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @nisarga-kh but you need to make the following changes:

1. import `_rotate-anticlockwise.less` in `animated.less`
2. Run `npm run build`

After you do that please commit the changes so that we can see them on the website

**Comment by nisarga-kh** [comment]
@shahednasser Have done the necessary modifications and have committed. Please do review once

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @nisarga-kh for making the changes but the problem with this is that you're using in the animation the `top` and `left` attributes, but this button isn't absolute and it shouldn't be.

I suggest you use translate in the animation instead.

**Pull request closed without merging**